src/act.comm.cpp

```diff
@@ -124,7 +124,7 @@ ACMD(do_say)
         act(buf, FALSE, ch, NULL, NULL, TO_ROOM);
       } else {
         success = success_test(GET_SKILL(ch, GET_LANGUAGE(ch)), 4);
-        for (tmp = world[ch->in_room].people; tmp; tmp = tmp->next_in_room)
+        for (tmp = world[get_absolute_in_room(ch)].people; tmp; tmp = tmp->next_in_room)
```

Comment:
This actually makes it so that when you speak inside a vehicle it's heard outside normally. PR denied.